docs/modules/ROOT/pages/extensions/componentdsl.adoc

```diff
@@ -0,0 +1,22 @@
+// Do not edit directly!
+// This file was generated by camel-quarkus-package-maven-plugin:update-extension-doc-page
+
+[[componentdsl]]
+= Component DSL
+
+[.badges]
+[.badge-key]##Since Camel Quarkus##[.badge-version]##1.0.0-M3## [.badge-key]##JVM##[.badge-supported]##supported## [.badge-key]##Native##[.badge-supported]##supported##
+
+Create Camel component with a fluent Java DSL
```

Comment:
The description in the runtime module is different. Could you please regenerate?

Reply:
I think the PR job should fail if it ends up generating/amending files